source/extensions/filters/http/local_ratelimit/local_ratelimit.h

```diff
@@ -85,6 +96,7 @@ class FilterConfig : public Router::RouteSpecificFilterConfig {
   Router::HeaderParserPtr response_headers_parser_;
   const uint64_t stage_;
   const bool has_descriptors_;
+  const envoy::extensions::filters::http::local_ratelimit::v3::LocalRateLimit& proto_config_;
```

Comment:
I think we need to copy the proto object, since this reference could (will) go stale.